javaagent-log-appender/README.md

```diff
@@ -0,0 +1,54 @@
+# OpenTelemetry Log Appender built-in OpenTelemetry Java Agent Example
+
+This example demonstrates an application configured to use the OpenTelemetry Log
+Appenders built-in OpenTelemetry Java Agent to bridge logs into the OpenTelemetry Log SDK, and export
+via [OTLP](https://opentelemetry.io/docs/reference/specification/protocol/otlp/).
+
+Details about the example:
+
+* The OpenTelemetry Log SDK is configured to export data to
+  the [collector](https://opentelemetry.io/docs/collector/), which prints the
+  logs to the console.
+* The application is configured with a variety of log solutions:
+  * Log4j API [configured](./src/main/resources/log4j2.xml) to print logs to the
+    console and
+    the [OpenTelemetry Log4J Appender](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/log4j/log4j-appender-2.17/javaagent/README.md).
```

Comment:
I don't follow what it means.
Is it needed?

Reply:
Yes, it is needed, I have clarified it further.